sdk/formrecognizer/ai-form-recognizer/recordings/node/aad_formrecognizerclient_nodejs_only/recording_recognizes_content_from_a_pdf_file_stream.js

```diff
@@ -43,138 +43,91 @@ nock('https://endpoint:443', {"encodedQueryParams":true})
   'Content-Length',
   '0',
   'Operation-Location',
-  'https://endpoint/formrecognizer/v2.0/layout/analyzeResults/e7de431a-1892-4270-b496-8cf2d55200ac',
+  'https://endpoint/formrecognizer/v2.0/layout/analyzeResults/79b2b7b5-419b-47f2-8f5e-8bcb44491440',
   'x-envoy-upstream-service-time',
-  '62',
+  '145',
   'apim-request-id',
-  'e7de431a-1892-4270-b496-8cf2d55200ac',
+  '79b2b7b5-419b-47f2-8f5e-8bcb44491440',
   'Strict-Transport-Security',
   'max-age=31536000; includeSubDomains; preload',
   'x-content-type-options',
   'nosniff',
   'Date',
-  'Tue, 18 Aug 2020 23:52:08 GMT'
-]);
-
-nock('https://login.microsoftonline.com:443', {"encodedQueryParams":true})
```

Comment:
This deletion is an example of the request that was saved because the token refresher wasn't shared.